twitchpopups.htm

```diff
@@ -92,14 +100,14 @@ <h1 id="popuptext"></h1>
     function onMessageHandler(target, context, msg, self) {
         // Remove whitespace from chat message
         const commandName = msg.trim();
-        
+        const formattedMessage = formatEmotes(commandName, context.emotes)
         // If the command is known, let's execute it
 
             if (context.mod || (context["badges-raw"] != null && context["badges-raw"].startsWith("broadcaster"))) {
                 if (commandName.startsWith("!alert ")) {
                     $("#popupbox").show();
                     $("#popupbox").css({"background-color":alertBg}); 
-                    $("#popuptext").text(commandName.substr(7).toUpperCase());
+                    $("#popuptext").html(formattedMessage.substr(7));

```

Comment:
This was changed early on from .html() to .text() as html was allowing users to say HTML tags in chat, e.g. `<img src='https://placehold.it/200x200' />`

Just a heads up we'll need to sanitise HTML if this goes through.

Reply:
the new function "formatEmotes()" DOES sanitise the html by using text() first and THEN returns a string containing <img> tags (which are intentionally not sanitised).

No change necessary.

Reply:
Ah nice! My apologies :)